dev/rest.rst

```diff
@@ -32,9 +32,21 @@ Config Endpoints
 
 .. toctree::
    :maxdepth: 1
+   :glob:
```

Comment:
There's no globbing here, is there?

Reply:
Just copied from the other subsections.  No idea if it's needed at all.

Reply:
`/home/acolomb/src/syncthing-docs/dev/rest.rst:44: WARNING: toctree contains reference to nonexisting document 'rest/cluster-*'`

No, it's needed.

Reply:
This is the "Config Endpoints" section though, not cluster. There is globbing in cluster (and everywhere else), but not in config.

Reply:
Sorry, just a merge gone bad then. Fixed.